app/editor/src/features/content/form/ContentClipForm.tsx

```diff
@@ -0,0 +1,248 @@
+import { Modal } from 'components/modal';
+import { useFormikContext } from 'formik';
+import { IFolderModel, IItemModel } from 'hooks/api-editor';
+import { useModal } from 'hooks/modal';
+import React from 'react';
+import { toast } from 'react-toastify';
+import { Button, ButtonVariant, Col, Row, Text } from 'tno-core';
+
+import { useStorage } from '../../../store/hooks';
+import { useContent } from '../../../store/hooks';
+import { defaultFolder } from '../../storage/constants';
+import { ClipDirectoryTable } from './ClipDirectoryTable';
+import { IContentForm } from './interfaces';
+import * as styled from './styled';
+
+export interface IContentClipFormProps {
+  content: IContentForm;
+}
+
+/**
+ * The component to be displayed when the clips tab is selected from the content form.
+ * @returns the ContentClipForm
+ */
+export const ContentClipForm: React.FC<IContentClipFormProps> = ({ content }) => {
+  const videoRef = React.useRef<HTMLVideoElement>(null);
+
+  const [path, setPath] = React.useState('/' + content.source);
+  const [folder, setFolder] = React.useState<IFolderModel>(defaultFolder);
+  const [streamUrl, setStreamUrl] = React.useState<string>();
+  const [item, setItem] = React.useState<IItemModel>();
+  const [file, setFile] = React.useState<File>();
+  const [start, setStart] = React.useState<string>('');
+  const [end, setEnd] = React.useState<string>('');
+  const [clipNbr, setClipNbr] = React.useState<number>(1);
+  const [currFile, setCurrFile] = React.useState<string>('');
+  const [target, setTarget] = React.useState<string>('');
+  const { toggle, isShowing } = useModal();
+  const storage = useStorage();
+  const { values, setFieldValue } = useFormikContext<IContentForm>();
+
+  React.useEffect(() => {
+    storage.getFolder(path).then((data) => {
+      setFolder(data);
+    });
+  }, [path, storage]);
+
+  React.useEffect(() => {
+    if (!!streamUrl && !!videoRef.current) {
+      videoRef.current.src = streamUrl;
+    }
+  }, [streamUrl, videoRef]);
+
+  const deleteItem = async (item: IItemModel) => {
+    try {
+      setItem(item);
+      toggle();
+    } catch {
+      // Ignore error a toast would have already been displayed with the error.
```

Comment:
Why kind of errors are you expected by opening the dialog box?

Reply:
I just copied this from your StorageListView and didn't think twice about that comment or which errors might be caught :-/

Reply:
See what happens when you remove this stuff.  I don't see any reason an error would be thrown here.

Reply:
Removed it.